toonz/sources/toonzqt/styleeditor.cpp

```diff
@@ -1691,6 +1698,81 @@ void PlainColorPage::setColor(const TColorStyle &style,
 }
 
 //-----------------------------------------------------------------------------
+
+void PlainColorPage::setVisibleParts(int settings) {
+  m_visibleParts = settings;
+  if (m_visibleParts & 0x01)
+    m_wheelShowButton->setChecked(true);
+  else
+    m_wheelShowButton->setChecked(false);
+  if (m_visibleParts & 0x02)
+    m_hsvShowButton->setChecked(true);
+  else
+    m_hsvShowButton->setChecked(false);
+  if (m_visibleParts & 0x04)
+    m_alphaShowButton->setChecked(true);
+  else
+    m_alphaShowButton->setChecked(false);
+  if (m_visibleParts & 0x08)
+    m_rgbShowButton->setChecked(true);
+  else
+    m_rgbShowButton->setChecked(false);
+}
+
+//-----------------------------------------------------------------------------
+
+int PlainColorPage::getVisibleParts() {
+  int visibleParts = 0;
+  if (m_wheelShowButton->isChecked()) visibleParts |= 0x01;
+  if (m_hsvShowButton->isChecked()) visibleParts |= 0x02;
+  if (m_alphaShowButton->isChecked()) visibleParts |= 0x04;
+  if (m_rgbShowButton->isChecked()) visibleParts |= 0x08;
+  return visibleParts;
+}
+
+//-----------------------------------------------------------------------------
+
+void PlainColorPage::setIsVertical(bool isVertical) {
+  // if (m_isVertical == isVertical) return;
+  // not returning even if it already is the same orientation
+  // to take advantage of the resizing here
+  // this is useful for the first time the splitter is set
+  // afterwards, it will be overridden by the saved state
+  // from settings.
+  m_isVertical = isVertical;
+  if (isVertical) {
+    m_vSplitter->setOrientation(Qt::Vertical);
+    m_toggleOrientationButton->setText("↔");
```

Comment:
Sorry for my late reply, but can you please replace this "↔" by `QChar(0x2194)` as well?